source/renderer/app/components/staking/layouts/StakingWithNavigation.js

```diff
@@ -10,6 +10,7 @@ type Props = {
   activeItem: string,
   onNavItemClick: Function,
   isActiveNavItem: Function,
+  isIncentivizedTestnet?: boolean,
```

Comment:
@yakovkaravelov this one can't be `undefined`

Reply:
Resolved @tomothespian @yakovkaravelov